src/test/common/steps/cookie-steps.js

```diff
@@ -13,6 +21,23 @@ Then(/^all page content is present on the cookies page$/, async function () {
   await checkAllCookiePageContentIsPresentAndCorrect()
 })
 
+Then(/^no back link is shown$/, async function () {
+  try {
+    console.log('Timeout error for getting back link expected')
+    await pages.cookies.getBackLink()
+    assert.fail('Backlink found on page but was not expected')
+  } catch (error) {
+    // no nothing, we expect there not to be a back link.
+    return true
+  }
+})
+
+Then(/^the back link on the cookies page links to the enter name page$/, async function () {
+  const backLink = await pages.cookies.getBackLink()
+  const href = await backLink.getAttribute('href')
+  expect(href).to.equal(`${pages.url}/enter-name`)
```

Comment:
I think this URL should be a constant in the `EnterName` page object as it is already defined and used in there, so safer to match via a constant.

Reply:
Done.

Reply:
Done.